src/blocks/HeroBlock/HeroBlock.tsx

```diff
@@ -0,0 +1,246 @@
+import { css, keyframes } from '@emotion/core'
+import styled from '@emotion/styled'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+  MinimalColorComponent,
+} from 'blocks/BaseBlockProps'
+import {
+  ButtonLinkBrandPivot,
+  ButtonStyleType,
+} from 'components/ButtonBrandPivot/Button'
+import { FontSizes, Heading } from 'components/Heading/Heading'
+import { HedvigH } from 'components/icons/HedvigH'
+import React from 'react'
+import { TextPosition } from 'src/utils/textPosition'
+import { LinkComponent } from 'storyblok/StoryContainer'
+import { getStoryblokImage, getStoryblokLinkUrl, Image } from 'utils/storyblok'
+import {
+  ContentWrapper,
+  getMinimalColorStyles,
+  MOBILE_BP_UP,
+  SectionWrapper,
+  TABLET_BP_UP,
+} from '../../components/blockHelpers'
+
+interface Animatable {
+  animate?: boolean
+}
+
+type TextProps = {
+  colorComponent?: MinimalColorComponent
+  textPosition?: TextPosition
+} & Animatable
+
+interface WrapperProps {
+  colorComponent?: MinimalColorComponent
+  height?: '80vh' | '90vh' | '100vh'
+}
+
+const Wrapper = styled(SectionWrapper)<WrapperProps>`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  min-height: ${(props) => props.height ?? '100vh'};
+  padding-top: 2.5rem;
+  padding-bottom: 2.5rem;
+
+  ${TABLET_BP_UP} {
+    padding-bottom: 3.75rem;
+  }
+`
+
+const WrapperWithExtraStyling = styled(Wrapper)<{ extraStyling?: string }>`
+  ${({ extraStyling }) => extraStyling};
+`
+
+const fadeSlideIn = keyframes`
+  from {
+    opacity: 0; 
+    transform: translateY(10%); 
+  }
+  to {
+    opacity: 1; 
+    transform: translateY(0%);
+  }
+`
+
+const HeroContent = styled(ContentWrapper)`
+  z-index: 2;
+`
+
+const HeroHeadline = styled(Heading)<Animatable>`
+  position: relative;
+  animation: ${(props) =>
+    props.animate
+      ? css`
+          ${fadeSlideIn} 800ms forwards
+        `
+      : 'none'};
+  opacity: ${(props) => (props.animate ? 0 : 1)};
+  animation-delay: 1000ms;
+  margin-bottom: 1rem;
+
+  ${TABLET_BP_UP} {
+    margin-bottom: 1.25rem;
+  }
+
+  br {
+    display: none;
+    ${MOBILE_BP_UP} {
+      display: block;
+    }
+  }
```

Comment:
In order to have control over linebreaks in desktop and to let it flow in mobile

Reply:
hehe thank you, so simple but but solves the problem 😄